Update/chronicle mqtt (#789)
iotaledger/documentation: chronicle/1.1/references/data-model.md

- **Transaction:** Stores transactions by their hash

> - **Hints:** Provides secondary indexes to look up value transactions by specific fields

Reviewer: ```suggestion
- **Hints:** Stores the sharding information for a given vertex
```

![Transaction table](../images/tx-table.png)

## Hints table

> This table acts as a way to query value transactions by their fields. For example, selecting an address which has been used as an input or output within a time-range is done through the edge table.

Reviewer: ```suggestion
This table acts as a way to query the sharding information by transactions fields.
```

Reviewer: Okay I think we have to revisit the data-model.md file and update the diagrams and description
